### PR TITLE
Move modules to subdir

### PR DESCRIPTION
**Pull request by TheCodedOne**
Moves modules to a subdir, this makes no functional change purely an accessibility one for development

```
/base
/rendering
/resources
```
are now 
```
/modules/base
/modules/rendering
/modules/resources
```

### CODE REVIEW

**Comment by TheCodedOne** [comment]
While we probably could I feel like it overcomplicates building separate jars & dependency resolution

**Comment by MsRandom** [comment]
Fair, I think this is all good otherwise, since it makes it a lot cleaner, though maybe rename `base` to something like `core`?

**Comment by UpcraftLP** [comment]
you mean different source sets @MsRandom? tho I agree with coded on this one, submodules are the way to go.
Especially since some gradle stuff is hardcoded to the `main` source set.

File: settings.gradle [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
 include 'base'
+project(':base').projectDir = new File('./modules/base')
 
 include 'rendering'
+project(':rendering').projectDir = new File('./modules/rendering')
 
-include 'resources'
\ No newline at end of file
+include 'resources'
+project(':resources').projectDir = new File('./modules/resources')
```
Comment (UpcraftLP) [inline settings.gradle:10]: newline at the end of file for consistency

**Pull request merged**